var/spack/repos/builtin/packages/py-dnaio/package.py

```diff
@@ -13,6 +13,7 @@ class PyDnaio(PythonPackage):
     url      = "https://pypi.io/packages/source/d/dnaio/dnaio-0.3.tar.gz"
     git      = "https://github.com/marcelm/dnaio.git"
 
+    version('0.4.2', sha256='fa55a45bfd5d9272409b714158fb3a7de5dceac1034a0af84502c7f503ee84f8')
     version('0.3', sha256='47e4449affad0981978fe986684fc0d9c39736f05a157f6cf80e54dae0a92638')
 
     depends_on('python@3.4:', type=('build', 'run'))
```

Comment:
Can you add a build dep on `py-setuptools-scm`?

Reply:
Yes, just added a commit.